packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -301,6 +306,8 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
       pageElements.push(this._renderPage(pages[i]));
     }
 
+    // console.log(`Page elements ${pageElements.length}`);
```

Comment:
@micahgodbolt I assumed so.. :) They are handy when working with the list. That's also why I have some white space changes there, I uncommented them temporarily.